examples/gallery.rst

```diff
@@ -124,16 +114,4 @@ How to
 .. nbgallery::
     :name: howto
     :glob:
-    :reversed:
-
-    howto/*
```

Comment:
This doesn't look right and the how tos are gone in the preview: https://pymcio--361.org.readthedocs.build/projects/examples/en/361/gallery.html#how-to

Reply:
Just noticed lack of thumbnails. What is wrong with it?

Reply:
I think you deleted the `howto/*` with the latest merge from main somehow and so nothing is listed inside that "nbgallery"

Reply:
Ok. Will come at this again tomorrow with a fresh head.